README.md

```diff
@@ -2,4 +2,43 @@
 
 # Description
 
-Utilities for writing [sf](https://github.com/salesforcecli/cli) plugins.
+The @salesforce/sf-pluins-core provides utilities for writing [sf](https://github.com/salesforcecli/cli) plugins.
+
+## SfCommand Abstract Class
+
+The SfCommand abstract class extends @oclif/command class and adds useful extensions to ease the development of commands for use in the Salesforce Unified CLI.
```

Comment:
Maybe add a link to the command class: https://github.com/oclif/core/blob/main/src/command.ts